packages/react/src/components/Button/Button.tsx

```diff
@@ -65,15 +65,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
           className={classes.icon}
         />
       )}
-      {children && (
-        <Paragraph
-          as='span'
-          size={size}
-          className={classes.typography}
-        >
-          {children}
-        </Paragraph>
-      )}
+      {children}
```

Comment:
Remove unused `Paragraph` import

Reply:
he he, such things should be caught by eslint, no-unsued-imports. 😄

Reply:
It was  😉 Couldn't comment on it as the line was not changed in this PR 
![image](https://github.com/digdir/designsystem/assets/1070981/289a378a-e233-4f8f-bf93-43a1f21cc493)

Reply:
oh, I did not catch that! Great! :)

Reply:
pre-commit 🙈